recipe/meta.yaml

```diff
@@ -148,15 +148,15 @@ outputs:
   - name: _r-xgboost-mutex
     version: 2.0
     build:
-      string: cpu_0  # [cuda_compiler_version == "None"]
-      string: gpu_0  # [cuda_compiler_version != "None"]
+      string: rapidsai_cpu_0  # [cuda_compiler_version == "None"]
+      string: rapidsai_gpu_0  # [cuda_compiler_version != "None"]
 
   - name: r-xgboost
     script: install-r-xgboost.sh  # [not win]
     script: install-r-xgboost.bat  # [win]
     build:
-      string: cuda{{ cuda_compiler_version | replace('.', '') }}_r{{ r_base | replace('.', '') }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
-      string: cpu_r{{ r_base | replace('.', '') }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
+      string: rapidsai_cuda{{ cuda_compiler_version | replace('.', '') }}_r{{ r_base | replace('.', '') }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+      string: rapidsai_cpu_r{{ r_base | replace('.', '') }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
```

Comment:
Same question as with `python-xgboost`

Reply:
Are the R packages also using some kind of CFFI solution for interacting with `libxgboost`?

Reply:
The R package links directly with `libxgboost` through the C layer